README.md

```diff
@@ -1,8 +1,8 @@
 # d3-dispatch
 
-Dispatching is a convenient mechanism for separating concerns with loosely-coupled code: register named callbacks and then call them with arbitrary arguments. A variety of D3 components, such as [d3-drag](https://github.com/d3/d3-drag), use this mechanism to emit events to listeners. Think of this like Node’s [EventEmitter](https://nodejs.org/api/events.html), except every listener has a well-defined name so it’s easy to remove or replace them.
+Dispatching is a convenient mechanism for separating concerns with loosely-coupled code: create registry of custom event names, then attach callbacks to them, and then invoke the callbacks by an event name with arbitrary arguments. A variety of D3 components, such as [d3-drag](https://github.com/d3/d3-drag), use this mechanism to emit events to listeners. Think of this like Node’s [EventEmitter](https://nodejs.org/api/events.html), except every listener has a well-defined name so it’s easy to remove or replace them.
 
-For example, to create a dispatch for *start* and *end* events:
+For example, to define *start* and *end* events:
```

Comment:
I find recursive explanations particularly hard. Instead of explaining that dispatch is without using the word "dispatch", the example just adds another layer to the stack of waiting to resolve the meaning of this word.